File: selectionSort.js
const selectionSort = (array) => {
	let idx = 0
	while (idx < array.length - 1) {
		let lowest = idx
		for (let i = idx + 1; i < array.length; i++) {
			if (array[lowest] > array[i]) {
				lowest = i
			}
		}
		let temp = array[lowest]
		array[lowest] = array[idx]
		array[idx] = temp
	    idx++
    }
	return array
}

File: selectionSort.py
def selectionSort(array):
    startIdx = 0
    while startIdx < len(array) - 1:
        lowest = startIdx
        for i in range(startIdx + 1, len(array)):
            if array[lowest] > array[i]:
                lowest = i
        swap(array, startIdx, lowest)
        startIdx += 1
    return array

def swap(array, idx1, idx2):
    array[idx1], array[idx2] = array[idx2], array[idx1]    
